[2024-06-11T21:45:15.221+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-11T21:45:15.283+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_702974 manual__2024-06-11T21:41:03.459046+00:00 [queued]>
[2024-06-11T21:45:15.303+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_data_mb.load_index_id_702974 manual__2024-06-11T21:41:03.459046+00:00 [queued]>
[2024-06-11T21:45:15.304+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-06-11T21:45:15.331+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): load_index_id_702974> on 2024-06-11 21:41:03.459046+00:00
[2024-06-11T21:45:15.340+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'load_data_mb', 'load_index_id_702974', 'manual__2024-06-11T21:41:03.459046+00:00', '--job-id', '270', '--raw', '--subdir', 'DAGS_FOLDER/load_data.py', '--cfg-path', '/tmp/tmp6ld1hrpe']
[2024-06-11T21:45:15.344+0000] {standard_task_runner.py:91} INFO - Job 270: Subtask load_index_id_702974
[2024-06-11T21:45:15.345+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=204) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-06-11T21:45:15.348+0000] {standard_task_runner.py:63} INFO - Started process 205 to run task
[2024-06-11T21:45:15.427+0000] {task_command.py:426} INFO - Running <TaskInstance: load_data_mb.load_index_id_702974 manual__2024-06-11T21:41:03.459046+00:00 [running]> on host f7fd8afd804e
[2024-06-11T21:45:15.733+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='load_data_mb' AIRFLOW_CTX_TASK_ID='load_index_id_702974' AIRFLOW_CTX_EXECUTION_DATE='2024-06-11T21:41:03.459046+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-11T21:41:03.459046+00:00'
[2024-06-11T21:45:15.735+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-11T21:45:15.820+0000] {logging_mixin.py:188} INFO - Ошибка в коде или неверный json
[2024-06-11T21:45:15.821+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-11T21:45:15.822+0000] {taskinstance.py:2892} ERROR - Task failed due to SystemExit(1)
[2024-06-11T21:45:15.898+0000] {taskinstance.py:1205} INFO - Marking task as FAILED. dag_id=load_data_mb, task_id=load_index_id_702974, execution_date=20240611T214103, start_date=20240611T214515, end_date=20240611T214515
[2024-06-11T21:45:16.005+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 270 for task load_index_id_702974 (Task failed due to SystemExit(1); 205)
[2024-06-11T21:45:16.048+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-11T21:45:16.101+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-11T21:45:16.107+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
